Makefile

```diff
@@ -9,6 +9,7 @@ DOCKER_CACHE_ORG ?=
 
 GO_TEST_FLAGS ?= -race -count=1
 
+BUF ?= buf
```

Comment:
By adding these you can quickly test the makefile by running `make push BUF="echo buf" DOCKER="echo docker"` to see the commands that would be executed.